MvvmCross/IoC/MvxIoCContainer.cs

```diff
@@ -584,13 +606,17 @@ protected virtual void InjectProperties(object toReturn)
             _propertyInjector?.Inject(toReturn, _options.PropertyInjectorOptions);
         }
 
-        protected virtual List<object> GetIoCParameterValues(Type type, ConstructorInfo firstConstructor)
+        protected virtual List<object> GetIoCParameterValues(Type type, ConstructorInfo firstConstructor, IDictionary<string, object> arguments)
```

Comment:
Should `arguments`be `arguments  = null`?

Reply:
I don't think it matters at all.

Reply:
Internal method. `arguments` is always passed.